### DISCUSSION TITLE
The code is not highlighted after build

### DISCUSSION THREAD

**Discussion by Jess-Qiu** [web-infra-dev/rspress#726] [Q&A]
![image](https://github.com/web-infra-dev/rspress/assets/22249167/4481942c-65eb-4cc4-9798-ebd3b1f98a89)

**Comment by sanyuan0704** [comment]
升级到 1.13.2 版本看看

**Reply by Jess-Qiu** [reply]
你好，我这边按照文档重新构建了新项目. build 之后 preview,发现只有 JavaScript 正常高亮

```json
{
  "name": "rspress-doc-template",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "rspress dev",
    "build": "rspress build",
    "preview": "rspress preview"
  },
  "dependencies": {
    "rspress": "^1.13.2"
  },
  "devDependencies": {
    "@types/node": "^16"
  }
}
```

![image](https://github.com/web-infra-dev/rspress/assets/22249167/9c2cdff6-add8-46f3-b7e3-bc6544110a00)
